src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -728,3 +1314,11 @@ const MCHAR* AppleseedRendererClassDesc::InternalName()
     // Parsable name used by MAXScript.
     return L"appleseedRenderer";
 }
+
+const MCHAR* AppleseedRendererClassDesc::GetRsrcString(INT_PTR id)
+{
+    const asf::KeyValuePair<int, const wchar_t*>* dialog_string_pair =
```

Comment:
This is where C++11's `auto` comes handy:

    const auto dialog_string_pair = LOOKUP_KVPAIR_ARRAY(g_dialog_strings, id);